### DISCUSSION TITLE
Suggestion for further precision with objects in the editor

### DISCUSSION THREAD

**Discussion by furiner** [ppy/osu#23806] [Ideas] [answered]
as mentioned on discord

on stable the closest you can place an object next to another before it automatically stacks the two is here
![](https://cdn.discordapp.com/attachments/1097318920991559880/1115934946167631893/osu_AulhoepEAi.png)

but on lazer it's around here
![](https://cdn.discordapp.com/attachments/1097318920991559880/1115934734636290058/osu_NofWVtg8EK.png)

the best usecase for this is more precision with manually placing streams, near-stacked doubles/triples or other similar forms of object pattern

so i was wondering if we can get that same level of precision on lazer aswell

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/23812
